utilities/output_tools.py

```diff
@@ -36,6 +36,14 @@ def metaInfoDict(exclude_diff='notebooks'):
     return meta_data
 
 def write_analysis_output(results, outfile, postfix):
+    print()
+    print("Unweighted events (before cut)")
+    print("-"*30)
+    for key in results.keys():
+        print(f"Dataset {key.ljust(30)}:  {results[key]['event_count']}")
+        print("-"*30)
+    print()
+
```

Comment:
Can you make this be output by logging.debug and only printed if you run with --debug? I hate the super noisy output

Reply:
ok, I actually wanted to have it as info because I always wanted that, but debug is fine too